### DISCUSSION TITLE
Dell U2720Q increase brightness past max display goes away hard

### DISCUSSION THREAD

**Discussion by jboxman** [MonitorControl/MonitorControl#1176] [General]
I have a U2720Q attached via HDMI; The hardware adjustment doesn't seem to happen, but the software brightness works. When the OSD was at max, I tapped the keyboard brightness key once more, and my display went away. It wouldn't come back until I switch to USBc. Fortunately, it doesn't seem broken.

This probably isn't expected behavior? What to do?

Thanks.

Oh, so HDMI no longer works at all. Is there some way to recover the use of my HDMI input? Or is this just gone now?

Had to reboot.

**Comment by waydabber** [comment]
The HDMI port of recent macs do not pass through DDC properly #750. Sometimes something goes through though which tend to drive some displays crazy. You must use USB-C if you want to use DDC.

**Reply by jboxman** [reply]
What I found strange is that, the monitor worked when plugged into a different Mac; Cycling the monitor didn't help, only rebooting the affected Mac did. Seems like I confused the Mac Mini hardware itself?
